components/GameForm/GameForm.js

```diff
@@ -74,12 +83,20 @@ export default function AddGameForm() {
         <InputContainer
           role="input"
           type="text"
-          // minLength="2"
           required
           name="opponent"
           id="opponent"
           aria-label="name"
         />
+        <FormLabel htmlFor="image">Upload Image</FormLabel>
+        <InputImageContainer
+          accept=".jpg, .jpeg, .png"
```

Comment:
Small hint: If you only accept a specific type of images, you could add a info-text for that (better user experience).